internal/model/job.go

```diff
@@ -64,6 +66,11 @@ type ExistingPR struct {
 	DependencyVersion string `json:"dependency-version" yaml:"dependency-version"`
 }
 
+type ExistingGroupPR struct {
+	DependencyGroupName string       `json:"dependency-group-name" yaml:"dependency-group-name"`
+	Dependencies        []ExistingPR `json:"dependencies" yaml:"dependencies"`
```

Comment:
This feels a little funny, I wasn't sure what to rename `ExistingPR` to as it isn't a Dependency object and anything I could think of was super verbose, so I left this as-is for now.